projects/fal/src/fal/exceptions/_base.py

```diff
@@ -3,15 +3,4 @@
 
 class FalServerlessException(Exception):
     """Base exception type for fal Serverless related flows and APIs."""
-
-    message: str
-
-    hint: str | None
-
-    def __init__(self, message: str, hint: str | None = None) -> None:
-        self.message = message
-        self.hint = hint
-        super().__init__(message)
-
-    def __str__(self) -> str:
-        return self.message + (f"\nHint: {self.hint}" if self.hint else "")
```

Comment:
base exception should as basic as possible. We don't really need to invent message/hint convention here yet.